src/main/java/edu/KevinChen/advancedjava/assignment2/IntervalEnum.java

```diff
@@ -0,0 +1,12 @@
+package edu.KevinChen.advancedjava.assignment2;
+
+/**
+ * Declares a set of enum values for IntervalEnum
+ */
+
+
+public enum IntervalEnum {
+
+    Daily, Monthly, yearly;
```

Comment:
this is probably not fine grain enough 
minutes, hours, days, weeks 
might be better